disk/tests/utils.py

```diff
@@ -8,3 +8,8 @@
 requires_linux = pytest.mark.skipif(ON_MACOS or ON_WINDOWS, reason='Requires Linux')
 requires_unix = pytest.mark.skipif(ON_WINDOWS, reason='Requires Linux or macOS')
 requires_windows = pytest.mark.skipif(not ON_WINDOWS, reason='Requires Windows')
+
+
+def assert_regex_equal(a, b):
+    assert a.pattern == b.pattern
+    assert a.flags == b.flags
```

Comment:
(nit) `assert regex_a.pattern, regex_a.flags == regex_b.pattern, regex_b.flags` will give a better assertion output in case of failure